core/scss/colors/settings-base.scss

```diff
@@ -4,7 +4,7 @@
 // @momentum-ui/core/tokens/colors.json
 //------------------------------------------------------
 
-$md-theme-5: #EBFCFF !default;
+$md-theme-05: #EBFCFF !default;
```

Comment:
after running `yarn build`, a few things got updated in `/core`. Should I remove these changes or keep them? @aimeex3 @bfbiggs

Reply:
looks like these are merge conflicts. looks like you also need to rebase. i would keep what's in master. it looks like the -05 is defined after the -95 in master